
def validate_data_set_file(file):
  errors = []
  return errors
